### PR TITLE
Update documents

### PR DESCRIPTION
**Pull request by Chuhaa**
$subject

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +11,10 @@ The `microsoft.excel` is a [Ballerina](https://ballerina.io/) connector for Micr
 
 This connector provides operations for connecting and interacting with Microsoft Graph API Excel endpoints over the network. 
 For more information about configuration and operations, go to the module. 
-- [`microsoft.excel`](excel/Module.md)
+- [`ballerinax/microsoft.excel`](excel/Module.md)
 
-# Building from the Source
-## Setting Up the Prerequisites
+# Building from the source
```
Comment (shaniR) [inline README.md:16]: Heading level should be ##

File: README.md [hunk 2]
```diff
@@ -11,10 +11,10 @@ The `microsoft.excel` is a [Ballerina](https://ballerina.io/) connector for Micr
...
+## Setting up the prerequisites
```
Comment (shaniR) [inline README.md:17]: Heading level should be ###

File: README.md [hunk 3]
```diff
@@ -42,10 +42,10 @@ As an open source project, Ballerina welcomes contributions from the community.
 
 For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/main/CONTRIBUTING.md).
 
-# Code of Conduct
+# Code of conduct
```
Comment (shaniR) [inline README.md:45]: Recheck heading levels with the guideline docs.

File: excel/Module.md [hunk 4]
```diff
@@ -1,89 +1,23 @@
...
-#### Step 1: Import Excel module
+### Step 1: Import connector
 First, import the ballerinax/microsoft.excel module into the Ballerina project.
 ```ballerina
 import ballerinax/microsoft.excel;
 ```
-#### Step 2: Configure the connection credentials.
-You can now make the connection configuration using the OAuth2 refresh token grant config.
+### Step 2: Create a new connector instance
+Create a `excel:Configuration` with the OAuth2 tokens obtained, and initialize the connector with it.
```
Comment (shaniR) [inline excel/Module.md:20]: Create an

Comment (shaniR) [reply excel/Module.md:20]: Remove the ,

**Review by shaniR — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -11,10 +11,10 @@ The `microsoft.excel` is a [Ballerina](https://ballerina.io/) connector for Micr
```
[the remark this answers is not in the document]
Comment (Chuhaa) [reply README.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/pull/15/commits/3e16c3fcae95051a7c24ecef4a633a6e7f3e29ef

File: excel/Module.md [hunk 6]
```diff
@@ -1,89 +1,23 @@
 ## Overview
-Ballerina connector for Microsoft Excel is connecting the Excel API in Microsoft Graph via Ballerina language easily. It provides capability to perform perform CRUD (Create, Read, Update, and Delete) operations on [Excel workbooks](https://docs.microsoft.com/en-us/graph/api/resources/excel?view=graph-rest-1.0) stored in Microsoft OneDrive. 
+Ballerina connector for Microsoft Excel is connecting the  Microsoft Graph Excel API via Ballerina language. It provides capability to perform CRUD (Create, Read, Update, and Delete) operations on [Excel workbooks](https://docs.microsoft.com/en-us/graph/api/resources/excel?view=graph-rest-1.0) stored in Microsoft OneDrive.
```
[the remark this answers is not in the document]
Comment (Chuhaa) [reply excel/Module.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/pull/15/commits/3e16c3fcae95051a7c24ecef4a633a6e7f3e29ef

File: README.md (hunk 1 above)
Comment (Chuhaa) [reply README.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/pull/15/commits/3e16c3fcae95051a7c24ecef4a633a6e7f3e29ef

File: README.md (hunk 2 above)
Comment (Chuhaa) [reply README.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/pull/15/commits/3e16c3fcae95051a7c24ecef4a633a6e7f3e29ef

File: excel/Module.md (hunk 4 above)
Comment (Chuhaa) [reply excel/Module.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/pull/15/commits/3e16c3fcae95051a7c24ecef4a633a6e7f3e29ef

File: README.md (hunk 3 above)
Comment (Chuhaa) [reply README.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/pull/15/commits/3e16c3fcae95051a7c24ecef4a633a6e7f3e29ef

File: excel/Module.md [hunk 7]
```diff
@@ -1,89 +1,23 @@
...
-    refreshUrl = <REFRESH_URL>
-    refreshToken = <REFRESH_TOKEN>
-    workbookIdOrPath = <WORKBOOK_ID_OR_PATH>
-    ```
-
-    The `workbookIdOrPath` is workbook ID or file path (with the `.xlsx` extension from root. If you have a file in root directory with name of `Work.xlsx`, you need to pass `workbookIdOrPath` as `Work.xlsx`). Make sure you create a workbook in Microsoft OneDrive and pass the correct `workbookIdOrPath` before using the connector.
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+- Create [Microsoft Office365 account](https://signup.live.com/)
+- Obtain token - Follow [this link](https://docs.microsoft.com/en-us/graph/auth-v2-user#authentication-and-authorization-steps)
```
[the remark this answers is not in the document]
Comment (Chuhaa) [reply excel/Module.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/pull/15/commits/3e16c3fcae95051a7c24ecef4a633a6e7f3e29ef

File: excel/Module.md [hunk 8]
```diff
@@ -1,89 +1,23 @@
...
-    clientSecret = <CLIENT_SECRET>
-    refreshUrl = <REFRESH_URL>
-    refreshToken = <REFRESH_TOKEN>
-    workbookIdOrPath = <WORKBOOK_ID_OR_PATH>
-    ```
-
-    The `workbookIdOrPath` is workbook ID or file path (with the `.xlsx` extension from root. If you have a file in root directory with name of `Work.xlsx`, you need to pass `workbookIdOrPath` as `Work.xlsx`). Make sure you create a workbook in Microsoft OneDrive and pass the correct `workbookIdOrPath` before using the connector.
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+- Create [Microsoft Office365 account](https://signup.live.com/)
```
[the remark this answers is not in the document]
Comment (Chuhaa) [reply excel/Module.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/pull/15/commits/3e16c3fcae95051a7c24ecef4a633a6e7f3e29ef

**Review by abeykoon — APPROVED** [review]

File: excel/client.bal [hunk 9]
```diff
@@ -16,15 +16,21 @@
...
-    http:Client excelClient;
-
-    # Initializes the Excel connector client.
-    #
-    # + configuration - Configurations required to initialize the `Client`
-    # + return - An error on failure of initialization else `()`
+public isolated client class Client {
+    final http:Client excelClient;
+
+    # Initializes the connector. During initialization you can pass either http:BearerTokenConfig if you have a bearer
```
Comment (shaniR) [inline excel/client.bal:27]: To initialize, you can either pass the http:BearerTokenConfig if you have a bearer

File: excel/client.bal [hunk 10]
```diff
@@ -16,15 +16,21 @@
...
-
-    # Initializes the Excel connector client.
-    #
-    # + configuration - Configurations required to initialize the `Client`
-    # + return - An error on failure of initialization else `()`
+public isolated client class Client {
+    final http:Client excelClient;
+
+    # Initializes the connector. During initialization you can pass either http:BearerTokenConfig if you have a bearer 
+    # token or http:OAuth2RefreshTokenGrantConfig if you have OAuth tokens.
```
Comment (shaniR) [inline excel/client.bal:28]: token or the http:OAuth2RefreshTokenGrantConfig if you have OAuth 2.0 tokens.

**Pull request merged**